examples/Classification/pointcloud_classification.py

```diff
@@ -74,24 +91,28 @@
 
     with torch.no_grad():
         for idx, batch in enumerate(tqdm(val_loader)):
-            pred = model(batch[0])
-            loss = criterion(pred, batch[1].view(-1))
+            category = batch['attributes']['category'].cuda()
+            pred = model(batch['data'].cuda())
+            loss = criterion(pred, category.view(-1))
             val_loss += loss.item()
 
             # Compute accuracy
             pred_label = torch.argmax(pred, dim=1)
-            val_accuracy += torch.mean((pred_label == batch[1].view(-1)).float()).cpu().item()
+            val_accuracy += torch.mean((pred_label == category.view(-1)).float()).cpu().item()
             num_batches += 1
 
     print('Val loss:', val_loss / num_batches)
     print('Val accuracy:', val_accuracy / num_batches)
-
+end_time = time.time()
+print('Training time: {}'.format(end_time - start_time))
 test_loader = DataLoader(ModelNet(args.modelnet_root, categories=args.categories,
-                                  split='test', transform=transform, device=args.device),
-                         shuffle=True, batch_size=15)
+                                  split='test', transform=transform),
+                         shuffle=True, batch_size=15, num_workers=num_workers, pin_memory=pin_memory)
 
-test_batch, labels = next(iter(test_loader))
-preds = model(test_batch)
+test_batch = next(iter(test_loader))
+preds = model(test_batch['data'].cuda())
```

Comment:
`.to(device)` instead of `cuda()` (only if following an above comment asking for multi-device tests)

Reply:
Resolving this, as we aren't doing CPU tests now